operators/user/olm-installing-operators-in-namespace.adoc

```diff
@@ -26,7 +26,9 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 include::modules/olm-installing-from-operatorhub-using-cli.adoc[leveloffset=+1]
 .Additional resources
 
-* xref:../../operators/understanding/olm/olm-understanding-operatorgroups.adoc#olm-operatorgroups-about_olm-understanding-operatorgroups[About Operator groups]
+* See "Operator Lifecycle Manager concepts" for details about the following topics:
```

Comment:
Again not critical, but I don't love this intro instead of just having the two links. It could also just be the fact that this is in quotes instead of being a link itself.

Reply:
Was trying to use the more-helpful section headings as deep links instead of just repeating the assembly title twice. But I'll rework it.